### PR TITLE
add ECR guide and link to it from admin dash page

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/get-started/install/amazon-eks.mdx [hunk 1]
```diff
@@ -297,6 +297,72 @@ okteto-ingress-nginx-controller  LoadBalancer   10.0.7.73    a519c8b3b27f94...el
 
 You'll need to take the `EXTERNAL-IP` address (`a519c8b3b27f94...elb.amazonaws.com` in the example), and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name  `*`, pointing to the alias of the Elastic Load Balancer.
 
+### Configure a private registry with Amazon ECR
```
Comment (pchico83) [inline src/content/get-started/install/amazon-eks.mdx:300]: I think this creates additional friction when installing Okteto in AWS. Configuring ECR access is optional.

What I would have is a dedicate page in the `Administration` menu to talk about private registry configuration, and then, a section for ECR, Dockerhub, Google, Azure, ...

**Comment by codyjlandstrom** [comment]
@pchico83 Ready for review. I'll need to either spend some time next week or get some help updating our internal Google registry guide as they've moved from GCR to Artifact Registry

File: src/content/admin/registry-credentials/amazon-ecr.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: Amazon ECR
+description: Get Private Registry credentials from Amazon ECR 
+sidebar_label: Amazon ECR
+id: amazon-ecr
+---
+
+This tutorial will guide you on how to use a private registry in Amazon Web Services, its necessary credentials, and how to use it in Okteto.
```
Comment (pchico83) [inline src/content/admin/registry-credentials/amazon-ecr.mdx:8]:
```suggestion
This tutorial will guide you on how to use ECR private registry in Amazon Web Services with Okteto and how to configure its necessary credentials.
```

File: src/content/admin/registry-credentials/amazon-ecr.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+The steps to do it are: 
+
+1. Create User with access to ECR
+1. Get User Credentials
+1. Okteto config with credentials
+1. Add credentials to the Okteto Dashboard
+
+## Step 1: Create User with access to ECR
+
+Create IAM user with `aws cli` ([CLI Reference](https://docs.aws.amazon.com/cli/latest/reference/iam/create-user.html))
```
Comment (pchico83) [inline src/content/admin/registry-credentials/amazon-ecr.mdx:19]: Let's add `aws` as a requirement, as we do in the EKS guide:
https://www.okteto.com/docs/1.19/get-started/install/amazon-eks/#installation-requirements

File: src/content/admin/registry-credentials/amazon-ecr.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+```
+aws iam create-user --user-name private-registry-user
+
+aws iam attach-user-policy \
+--policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly \
+--user-name private-registry-user
+```
+
+Or, configure manually. Find the `IAM Service` in the **AWS Console,** and follow this steps:
```
Comment (pchico83) [inline src/content/admin/registry-credentials/amazon-ecr.mdx:29]:
```suggestion
Or, configure manually. Find the `IAM Service` in the **AWS Console,** and follow these steps:
```

File: src/content/admin/registry-credentials/amazon-ecr.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+- *Users → Create User*
+- Add User *Name*
+- Add *Policy Option*
+- Search for *ContainerRegistry* (for this example I’m adding **ReadOnly**)
+- Create User
+
+:::info
+It is more recommended to use groups, and add the user and policy to that group
```
Comment (pchico83) [inline src/content/admin/registry-credentials/amazon-ecr.mdx:38]:
```suggestion
It is recommended to use groups, and add the user and policy to that group
```

File: src/content/admin/registry-credentials/azure-acr.mdx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+This tutorial will guide you on how to use a private registry in **Azure Container Registry**, its necessary credentials, and how to use it in **Okteto**. 
+
+The steps to do it are: 
+
+1. Create User with access to ACR
+1. Get User Credentials
+1. Okteto config with credentials
+
+:::info
+Before getting started, make sure you have the [Azure CLI installed](https://learn.microsoft.com/en-us/cli/azure/)
```
Comment (pchico83) [inline src/content/admin/registry-credentials/azure-acr.mdx:17]: Let's list it as a requirement

File: src/content/admin/registry-credentials/registry-credentials.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/registry_credentials_view.png").default}
+    alt="Registry credentials view"
+    width="900"
+  />
+</p>
+
+When the `daemonset.configurePrivateRegistriesInNodes.enabled` setting is active, Okteto automatically applies registry credentials to nodes, allowing cluster-wide access to these registries.
```
Comment (pchico83) [inline src/content/admin/registry-credentials/registry-credentials.mdx:20]: @maroshii @codyjlandstrom could we document the new pull secret operator option and its advantages?

File: src/content/admin/registry-credentials/registry-credentials.mdx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+Registries are listed by type (Static or AWS) and hostname. Static credentials use a username and password, ideal for platforms like Docker Hub. [AWS credentials for ECR](https://docs.aws.amazon.com/ecr/index.html) use an Access Key and Secret, with Okteto refreshing credentials every 4 hours. For AWS, ensure the registry hostname ends with .amazonaws.com. Instructions are provided for retrieving ECR credentials.
+
+Follow one of the guides below to learn how to retrieve your registry credentials:
+- [Google Artifact Registry](admin/registry-credentials/google-artifact-registry.mdx)
```
Comment (pchico83) [inline src/content/admin/registry-credentials/registry-credentials.mdx:25]: Let's list them in alphabetical order

**Comment by mnevadom** [comment]
Looks good @codyjlandstrom !

Im going to update the GCP guide and will let you know as soon as possible!

**Comment by mnevadom** [comment]
@pchico83 Should the guide be just basics assuming migration from CR? 
Or should we add some tutorials/links for the migration?

**Comment by pchico83** [comment]
> @pchico83 Should the guide be just basics assuming migration from CR? Or should we add some tutorials/links for the migration?

@mnevadom yes, let's assume the user is already in Artifact Registry

File: src/content/admin/registry-credentials/registry-credentials.mdx (hunk 7 above)
Comment (maroshii) [reply src/content/admin/registry-credentials/registry-credentials.mdx:20]: I'll take care of the other pull secrets docs. I would merge this as-is and I'll update in the next PR

**Comment by mnevadom** [comment]
I already updated the guide: https://www.notion.so/okteto/Google-Cloud-GCR-477a2269d99b4ad8b4f7b07feb0a10f0?pvs=4

It has been tested in Product following the guide and it is working: 

image


image

File: src/content/admin/registry-credentials/registry-credentials.mdx (hunk 8 above)
Comment (codyjlandstrom) [reply src/content/admin/registry-credentials/registry-credentials.mdx]: Will do this in the sidebar as well

File: src/content/admin/dashboard.mdx [hunk 9]
```diff
@@ -207,39 +198,6 @@ From this page you can follow a simple step-by-step guide to get the major compo
 
 ## Registry Credentials
 
-In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. You can also manage these credentials directly in your Kubernetes cluster using the [CRDs](self-hosted/manage/crds.mdx#private-registries) created by Okteto.
-
-If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](self-hosted/helm-configuration.mdx#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes so the cluster can also pull images from those registries.
+If you choose to use your own registry service instead of [Okteto's Container Registry](core/container-registry.mdx), you can configure credentials for that service here.
```
Comment (pchico83) [inline src/content/admin/dashboard.mdx:201]: It's not one or the other. People must use Okteto Registry in any case. Private registries are to pull private images from other private registries.

File: src/content/admin/registry-credentials/google-artifact-registry.mdx [hunk 10]
```diff
@@ -0,0 +1,8 @@
+---
+title: Google Artifact Registry
+description: Get Private Registry credentials from Google Artifact Registry 
+sidebar_label: Google Artifact Registry
+id: google-artifact-registry
+---
+
+## Coming Soon ✨
```
Comment (pchico83) [inline src/content/admin/registry-credentials/google-artifact-registry.mdx:8]: I think mario did the new guide for Artifact Registry

**Review by pchico83 — CHANGES_REQUESTED** [review]

File: src/content/admin/dashboard.mdx (hunk 9 above)
Comment (codyjlandstrom) [reply src/content/admin/dashboard.mdx:201]: Oh I understand now. Let up update this copy

Comment (codyjlandstrom) [reply src/content/admin/dashboard.mdx]: Does this sound ok? `If you choose to pull images from a private registry in addition to [Okteto's Container Registry](core/container-registry.mdx), you can configure credentials for that service here.`

Comment (pchico83) [reply src/content/admin/dashboard.mdx]: LGTM

**Review by pchico83 — APPROVED** [review]

**Pull request merged**